aws/src/test/java/org/apache/iceberg/aws/glue/TestIcebergToGlueConverter.java

```diff
@@ -311,4 +312,81 @@ public void testSetTableDescription() {
+                    .columns(
+                        ImmutableList.of(
+                            Column.builder().name("x").comment("existing comment").build(),
+                            Column.builder().name("y").comment("existing comment").build()))
+                    .build())
+            .build();
+
+    IcebergToGlueConverter.setTableInputInformation(
+        actualTableInputBuilder, tableMetadata, existingGlueTable);
+    TableInput actualTableInput = actualTableInputBuilder.build();
+
+    // Expected TableInput
+    TableInput expectedTableInput =
+        TableInput.builder()
+            .storageDescriptor(
+                StorageDescriptor.builder()
+                    .location("s3://test")
+                    .additionalLocations(Sets.newHashSet(tableLocationProperties.values()))
+                    .columns(
+                        ImmutableList.of(
+                            Column.builder()
+                                .name("x")
+                                .type("string")
+                                .comment("existing comment")
+                                .parameters(
+                                    ImmutableMap.of(
+                                        IcebergToGlueConverter.ICEBERG_FIELD_ID, "1",
+                                        IcebergToGlueConverter.ICEBERG_FIELD_OPTIONAL, "false",
+                                        IcebergToGlueConverter.ICEBERG_FIELD_CURRENT, "true"))
+                                .build(),
+                            Column.builder()
+                                .name("y")
+                                .type("string")
+                                .comment("new comment")
+                                .parameters(
+                                    ImmutableMap.of(
+                                        IcebergToGlueConverter.ICEBERG_FIELD_ID, "2",
+                                        IcebergToGlueConverter.ICEBERG_FIELD_OPTIONAL, "false",
+                                        IcebergToGlueConverter.ICEBERG_FIELD_CURRENT, "true"))
+                                .build(),
+                            Column.builder()
+                                .name("z")
+                                .type("string")
+                                .comment("new comment")
+                                .parameters(
+                                    ImmutableMap.of(
+                                        IcebergToGlueConverter.ICEBERG_FIELD_ID, "3",
+                                        IcebergToGlueConverter.ICEBERG_FIELD_OPTIONAL, "false",
+                                        IcebergToGlueConverter.ICEBERG_FIELD_CURRENT, "true"))
+                                .build()))
+                    .build())
+            .build();
+
+    Assertions.assertThat(actualTableInput.storageDescriptor().columns())
+        .as("Columns should match")
+        .isEqualTo(expectedTableInput.storageDescriptor().columns());
```

Comment:
Double checking to make sure this test is strong, `equals` for columns will properly compare against all the fields of the column right? It may be better to explicitly assert the fields we want.

Reply:
Seems like it based on the unit test for testDropColumn, so we should be good.